config/jobs/kubernetes/gengo/gengo-config.yaml

```diff
@@ -52,3 +52,18 @@ presubmits:
             limits:
               memory: 4Gi
               cpu: 2
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230727-ea685f8747-master
```

Comment:
we usually recommend a single test container, prow has pretty limited support for multiple containers.

I would probably wrap both of these back in gengo repo with a single target comparable to `make verify` in k/k, though it can be a lot simpler (See e.g. the target in the KIND repo)

Reply:
that will also make it easier to add and enforce new linters by PRing the verify script/target in gengo (and ensure that new verify scripts are not enforced until they're passing in the PR that adds them, instead of cross-repo coordination)